scalafmt-core/shared/src/main/scala/org/scalafmt/config/Docstrings.scala

```diff
@@ -1,13 +1,48 @@
 package org.scalafmt.config
 
-import metaconfig.ConfCodec
+import metaconfig._
+import metaconfig.generic.Surface
 
-sealed abstract class Docstrings
+case class Docstrings(
+    oneline: Option[Docstrings.Oneline] = None,
```

Comment:
Does `oneline = None` mean «preserve»? May be just add `case object preserve extends Oneline`?

And what is purpose of replacing `Docstrings.preserve` with `style = None`?

Reply:
There used to be one option, and docstrings = preserve meant doing not modify formatting of docstrings.

Which means there's no formatting style, and any additional options (like oneline) should be ignored because formatting is disabled.

Ideally, since the entire thing is disabled, it should be null, or None, but since we haven't fixed metaconfig and have to put the decoder inside, it can't be null. So I'm setting style = None instead. But I don't want to allow the user to set docstrings.style = preserve either.

With oneline, i can add the third, default option.

Reply:
Ok, I figured out what is problem with style. In `oneline` third option will be good

Reply:
done